src/Database/Esqueleto/Record.hs

```diff
@@ -0,0 +1,423 @@
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TemplateHaskellQuotes #-}
+
+module Database.Esqueleto.Record
+  ( deriveEsqueletoRecord
+  , takeColumns
```

Comment:
`takeColumns` should be an internal implementation detail - we shouldn't need to export it.